lib/pbench/test/unit/server/query_apis/test_datasets_contents.py

```diff
@@ -454,7 +454,9 @@ def test_empty_query(
             self.elastic_endpoint,
             self.payload,
             index,
-            expected_status,
+            expected_status
+            if expected_status != HTTPStatus.OK
+            else HTTPStatus.NOT_FOUND,
```

Comment:
Well `black` certainly made *that* look just about as ugly as possible. I might be tempted to break the assignment out and change `expected_status` appropriately before making the call, but this ought to work.

Reply:
Yeah, that would have been better handled with a simple, separate `if` statement, before the function invocation...but I'd rather see this merged as is....